pkg/controller/complianceremediation/complianceremediation_controller.go

```diff
@@ -378,6 +395,123 @@ func (r *ReconcileComplianceRemediation) countKubeUnmetDependencies(rem *compv1a
 	return nMissingDeps, nil
 }
 
+func (r *ReconcileComplianceRemediation) handleValueRequired(rem *compv1alpha1.ComplianceRemediation, logger logr.Logger) (bool, reconcile.Result, error) {
+	annotations := rem.GetAnnotations()
+	labels := rem.GetLabels()
+	hasUpdate := false
```

Comment:
Same comment about potentially not needing the `hasUpdate` variable and not having to return the `reconcile.Result` as I had with `handleUnsetValues`

Reply:
I am going to fix it